cookbook/integrations/aws/athena/README.rst

```diff
@@ -1,7 +1,9 @@
+:nosearch:
```

Comment:
@katrogan I think we have this example right. can we add it?

Reply:
I have a very trivial one but support isn't out in an official flytekit release yet.